honeybot/plugins/join.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+"""
+[join.py]
+Join Plugin
+
+[Author]
+Marcelo Benesciutti
+
+[About]
+Will join a given channel in the server
+
+[Commands]
+>>> .join channel
+joins the given channel
+"""
+
+class Plugin:
+    def __init__(self):
+        pass
+
+    def run(self, incoming, methods, info):
+        try:
+            msgs = info['args'][1:][0].split()
+            raw_user = info['prefix']
+            user_index = raw_user.find('!')
+            user = raw_user[0:user_index]
+            f = open("settings/OWNERS.conf", "r")
```

Comment:
You should use the keyword "**with**" statement, this will make sure the file is closed after you have finished reading,
[Info Here](https://dbader.org/blog/python-context-managers-and-with-statement)
```suggestion
            with open("settings/OWNERS.conf") as f
```

Reply:
implemented the suggested change, thanks!